microsoft-dotnet/3.1/modules/ROOT/pages/microsoft-dotnet-connector-studio.adoc

```diff
@@ -0,0 +1,178 @@
+. On the *TLS* tab, optionally specify TLS information.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specific server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for Microsoft .NET, follow these steps:
+
+. In the *Mule Palette* view, select *Microsoft DotNet* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials the connector uses to access the target Microsoft .NET system.
+
+To configure the global element for Microsoft .NET Connector, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, complete these fields:
++
+image::microsoft-dotnet-connection.png[Studio General tab]
++
+* *Connection*
++
+Select a connection type.
++
+* *Scope*
++
+Select the connection scope:
++
+** *Singleton*
++
+Shares the same instance of the Microsoft .NET component across multiple calls. Use this option to store a shared state in class instance members across different calls.
+** *Transient*
++
+Creates a new instance of the Microsoft .NET component per request.
+* *Grant Full Trust to the .NET assembly*
++
+Microsoft .NET code can be executed in an application domain with restricted privileges to avoid the execution of malware code that can affect the stability of the Mule runtime engine. These restrictions include limited access to the file system, native code execution, network calls, registry access, and more. By default the Microsoft .NET connector is set to use full trust.
++
+* *Declared methods only*
++
+`true` is the default value. *Reviewers - can you provide a description for this field?*
+* *Include auto generated code*
++
+`false` is the default value. *Reviewers - can you provide a desciption for this field?*
++
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+=== Configure the External Connection Type
+
+Use the *External* connection type to connect with an assembly external to the application: (*Reviewers, please confirm that the previous text is correct.*)
```

Comment:
the previous is a `External Connection Type`

Reply:
It is correct. I was just saying that the example it is an External connection type. All good here.